packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/flow-diagram.md

```diff
@@ -2,48 +2,27 @@
+   ![Sequence diagram that displays the interaction between the resource owner, authorization server, and resource server for Authorization Code flow with PKCE](/img/authorization/oauth-auth-code-pkce-grant-flow.png)
 
+   <!--
+     Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?type=design&node-id=4133%3A43878&mode=design&t=Me7qqw8odOmrLh6K-1 oauth-auth-code-pkce-grant-flow
+   -->
 </div>
 
-<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
-
-@startuml
-skinparam monochrome true
-actor "Resource Owner (User)" as user
-participant "Client" as client
-participant "Authorization Server (Okta)" as okta
-participant "Resource Server (Your App)" as app
-
-autonumber "<b>#."
-client -> client: Generate PKCE code verifier & challenge
-client -> okta: Authorization Code request + code_challenge to /authorize
-okta -> user: 302 redirect to authentication prompt
-user -> okta: Authentication & consent
-okta -> client: Authorization Code response
-client -> okta: Send authorization code + code_verifier to /token
-okta -> okta: Evaluates PKCE code
-okta -> client: Access token (and optionally refresh token)
-client -> app: Request with access token
-app -> client: Response
-@enduml
-
--->
-
- At a high-level, the flow has the following steps:
+ At a high level, the flow has the following steps:
 
 1. Your application (app) generates a code verifier followed by a code challenge. See [Create the proof key for code exchange](#create-the-proof-key-for-code-exchange).
 2. Your app directs the browser to the Okta sign-in page, along with the generated code challenge.
 
-    You need to register your app so that Okta can accept the authorization request. See [Set up your app](#set-up-your-app) to register and configure your app with Okta. After registration, your app can redirect the browser to Okta. See [Request an authorization code](#request-an-authorization-code).
+   Before implementing this redirect request to the authorization server (Okta), you need to [set up your app](#set-up-your-app) in Okta. See [Request an authorization code](#request-an-authorization-code).
 
 3. The authorization server (Okta) redirects the authentication prompt to the user.
 4. The user authenticates.
 
     For Okta to authenticate the user credentials, Okta needs user profile data.
     See [Add a user using Console](https://help.okta.com/okta_help.htm?id=ext-usgp-add-users), [Import Users](/docs/guides/password-import-inline-hook/), and the [Users API](/docs/reference/api/users/). Alternatively, you can [set up self-service registration](/docs/guides/oie-embedded-sdk-use-case-self-reg/) to allow users to register their membership with the app.
 
-5. Okta redirects back to your native application with an authorization code.
+5. Okta redirects back to your mobile application with an authorization code.
```

Comment:
```suggestion
5. Okta redirects back to your platform-specific application with an authorization code.
```
Maybe safer to use "platform-specific" in this case since Auth code with PKCE is also used for SPA apps, not just mobile.

Reply:
Just checking if I can use "mobile app or SPA" instead of platform-specific. SPAs aren't platform-specific.

Reply:
Sure, sounds good.
Although you should also check with Fed team for this. In the past, devs have been adamant about using the word "Native app" (widely used industry term), so we try to update that adjective where we can. 
I've also heard that some web apps using Auth Code with PKCE flow too (when they can't secure the client secret or if they want additional security)